docs/building-a-form/README.md

```diff
@@ -0,0 +1,13 @@
+# Building a form
```

Comment:
Are these eventually going to be links to other pages?

Reply:
Yes, "Building a form" is a map topic that will act as a more constrained navigation page outlining the different aspects of... building a form. Same with "Getting started with the Forms System Library".

With this said, putting the "common patterns" (formerly cookbook/recipes) in their own separate map is definitely up for discussion. It could also go in a single topic under "Building a form". Any feelings on that?

Reply:
hm, good question. I don't have strong feelings so whatever you think is best!